hass/devices.js

```diff
@@ -212,6 +212,101 @@ module.exports = {
       }
     }
   ],
+  '5-1619-20549': [
+    // Intermatic PE653 MultiWave Receiver
+    {
+      type: 'climate',
+      object_id: 'pool_thermostat',
+      values: ['49-1-1', '67-1-1'],
+      default_setpoint: '67-1-1',
+      discovery_payload: {
+        min_temp: 40,
+        max_temp: 104,
+        modes: ['heat'],
+        temperature_unit: 'F',
+        current_temperature_topic: '49-1-1',
+        current_temperature_template: '{{ value_json.value }}',
+        temperature_command_topic: true,
+        temperature_state_template: '{{ value_json.value }}'
+      }
+    },
+    {
+      type: 'climate',
+      object_id: 'spa_thermostat',
```

Comment:
Same here. Then seems this are configured in the same way so you will have 2 thermostats that controls same values

Reply:
There are actually two thermostat instances, 67-1-1 and 67-1-7 but curiously only one water temperature sensor 49-1-1 that is used for both.  The thermostats have dedicated functions in the system.  One thermostat is used specifically for Pool temperature and the other for Spa temperature and they are labeled as such in the UI.

There's also support for air temperature and freeze sensors as well as some other auxiliary equipment I don't have, so that can be added by someone else later.  More information here if you're curious: https://www.intermatic.com/-/media/inriver/7092-8536.ashx